### PR TITLE
Fortune cookies counter

### PR DESCRIPTION
**Pull request by UladBohdan**

### CODE REVIEW

File: src/api/game/fortune/controller.js [hunk 1]
```diff
@@ -26,6 +28,10 @@ export const getCookie = ({ params: { slug } }, res, next) =>
     .then(checkIsFound)
     .then(fortuneGame => sample(fortuneGame.cookies))
     .then(formatFortuneCookie)
+    .then(async cookie => {
+      await Counter.inc(gameCounterKey(slug));
+      return cookie;
+    })
     .then(sendJson(res))
```
Comment (Drapegnik) [inline src/api/game/fortune/controller.js:35]: why not:
```
.then(formatFortuneCookie)
.then(sendJson(res))
.then(() => Counter.inc(gameCounterKey(slug)))
```

File: src/metrics/counter/model.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const joiCounterSchema = Joi.object({
+  key: Joi.string()
+    .required()
+    .meta({ unique: true }),
+  count: Joi.number().default(0),
+});
+
+const CounterSchema = joiToMongoose(joiCounterSchema);
+
```
Comment (Drapegnik) [inline src/metrics/counter/model.js:12]: please add some `get` api to view counters value

Comment (UladBohdan) [reply src/metrics/counter/model.js:12]: Done https://dev.wir.by/api/metrics/counter/game-fortune-ny2021

File: src/api/game/fortune/controller.js (hunk 1 above)
Comment (UladBohdan) [reply src/api/game/fortune/controller.js:35]: Does not work that way🤷

**Review by Drapegnik — APPROVED** [review]

**Pull request merged**